client/blocks/reader-full-post/index.jsx

```diff
@@ -500,6 +507,7 @@ export class FullPostView extends React.Component {
 								<EmbedContainer>
 									<AutoDirection>
 										<div
+											ref={ this.postContentWrapper }
```

Comment:
What about the `PostExcerpt` component above? Can it happen that the excerpt also contains embed iframes? If yes, it also deserves that the `ref` is passed, and the embeds are managed by the resize listener.

Reply:
Nope, `iFrames` are children of `figure` and are stripped from excerpts here https://github.com/Automattic/wp-calypso/blob/4abde5793955a3bb230acbc0515e3ad93741d6e2/client/lib/post-normalizer/rule-create-better-excerpt.js#L58-L60